bot.js

```diff
@@ -1406,29 +1426,55 @@ function command(message) {
             for (var i in Object.keys(config.customCommands.targeted)) {
                 var key = Object.keys(config.customCommands.targeted)[i];
 
-                if (
-                    message.content.startsWith(key) &&
-                    !config.customCommands.targeted[key].disabled
-                ) {
+                if (message.content.startsWith(key)) {
+                    let customCommand = config.customCommands.targeted[key];
+
                     log.info(
                         "Command " +
                             message.content +
                             " matched targeted custom command " +
                             key
                     );
 
-                    var operation = config.customCommands.targeted[key];
+                    if (
+                        !customCommand.disabled &&
+                        customCommand.alias != null
+                    ) {
+                        if (
+                            config.customCommands.targeted.hasOwnProperty(
+                                customCommand.alias
+                            )
+                        ) {
+                            customCommand =
+                                config.customCommands.targeted[
+                                    customCommand.alias
+                                ];
+                        } else {
+                            log.warning(
+                                "Targeted custom command " +
+                                    message.content +
+                                    " aliases " +
+                                    customCommand.alias +
+                                    ", which does not exist."
+                            );
+                            return;
+                        }
+
+                        // If the alias target is disabled, return
+                        if (customCommand.disabled) return;
+                    }
+
                     var output;
                     // Either random or format must be present. Prefer random if both exist.
-                    if (operation.random) {
-                        output = selectOne(operation.random);
+                    if (customCommand.random) {
```

Comment:
do you need another customCommand.disabled check here for when the non-alias command is used directly? It looks like you moved the disabled check for equalTo commands but not targeted, multiTargeted, and startsWith.

Reply:
Hm. It seems you are correct. Good catch.